deltacat/compute/converter/steps/convert.py

```diff
@@ -71,43 +115,34 @@ def filter_rows_to_be_deleted(
             data_file_table[identifier_column],
             equality_delete_table[identifier_column],
         )
-        positional_delete_table = data_file_table.filter(equality_deletes)
-        logger.info(f"positional_delete_table:{positional_delete_table.to_pydict()}")
+        position_delete_table = data_file_table.filter(equality_deletes)
+        logger.info(f"positional_delete_table:{position_delete_table.to_pydict()}")
```

Comment:
This seems like a pretty expensive operation for a logger message... can we either (1) replace this with something cheaper (like just a length of the position delete table) or (2) only execute lines like this if some debug/dev/trace flag is enabled?

Reply:
Yeah, these are actually debug logs, will change to table length to log out how many records we dropped in this step in follow-up PR